app/assets/stylesheets/suggest-list.css

```diff
@@ -0,0 +1,49 @@
+.suggest-list {
```

Comment:
Please remember to check mobile view
![Screenshot from 2019-08-12 18-39-46](https://user-images.githubusercontent.com/7680662/62867406-cef8c300-bd30-11e9-9ba8-84c0b53b9853.png)

Please make the list highlight (`.selected`) covers entire horizontal space of the dropdown, use `#e9573f` for highlighting color, add padding, bold font-weight and use round corners.

![Screenshot from 2019-08-12 18-42-33](https://user-images.githubusercontent.com/7680662/62867890-fbf9a580-bd31-11e9-800f-e2e1c13aa3fd.png)